notfound/utils.py

```diff
@@ -25,14 +34,36 @@ def replace_uris(app, doctree, nodetype, nodeattr):
             if uri.startswith('../'):
                 uri = uri.replace('../', '')
 
+        imagedir = ''
+        if nodetype == docutils.nodes.image:
```

Comment:
Is this a class? If so, you should use `is`.

Reply:
Done!